spec/services/govuk_emails/monitor_spec.rb

```diff
@@ -1,108 +1,181 @@
 require "rails_helper"
 
 RSpec.describe GovukEmails::Monitor do
-  let(:scheduled_mailing) { create(:scheduled_mailing, :processing) }
-  let(:response) { double GovukEmails::Email, status: }
-
   before do
     allow(GovukEmails::Email).to receive(:new).with(scheduled_mailing.govuk_message_id).and_return(response)
     Setting.setting.update(alert_via_sentry: true)
   end
 
+  let(:response) { double GovukEmails::Email, status: }
```

Comment:
Can we use `instance_double` here?

I'm surprised Rubocop didn't complain, although this file might be ignored!